d2game/d2gamescreen/main_menu.go

```diff
@@ -633,20 +633,16 @@ func (v *MainMenu) onBtnTCPIPOkClicked() {
 
 // getLocalIP returns local machine IP address
 func (v *MainMenu) getLocalIP() string {
-	conn, err := net.Dial("udp", "8.8.8.8:80")
+	host, _ := os.Hostname()
+	addrs, _ := net.LookupIP(host)
 
-	if err != nil {
-		v.logger.Error(err.Error())
-		return "cannot reach network"
-	}
-
-	err = conn.Close()
-	if err != nil {
-		v.logger.Error(err.Error())
-		return "unexpected error occurred while closing test connection"
+	for _, addr := range addrs {
+		if ipv4 := addr.To4(); ipv4 != nil {
+			return ipv4.String()
+		}
 	}
 
-	localAddr := conn.LocalAddr().(*net.UDPAddr)
+	v.logger.Error("no IPv4 Address could be found")
```

Comment:
I think, It should be LogLevelWarning

Reply:
Yeah. Why not. It's not a really bad error. But of course if you can't find any local interface than surely you won't be able to play over Network I guess.
I'll change it.